apps/framework-cli/src/utilities/capture.rs

```diff
@@ -57,39 +59,38 @@ macro_rules! capture {
         use chrono::Utc;
         use reqwest::Client;
         use serde_json::json;
+        use std::env;
+        use std::time::Duration;
         use uuid::Uuid;
 
-        #[allow(unused)]
-        let event = json!(MooseActivity {
-            id: Uuid::new_v4(),
-            project: $project_name,
-            activity_type: $activity_type,
-            sequence_id: $sequence_id,
-            timestamp: Utc::now(),
-            cli_version: constants::CLI_VERSION.to_string(),
-        });
-        let remote_url = {
-            let guard = PROJECT.lock().unwrap();
-            guard.instrumentation_config.url().clone()
-        };
-
-        // Sending this data can fail for a variety of reasons, so we don't want to
-        // block user & no need to handle the result
-        tokio::spawn(async move {
-            // TODO: Change this to MooseActivity after the table is verified
-            let instrumentation_url = format!("{}/ingest/UserActivity", remote_url);
-            // TODO: Delete this, use event from above instead
-            let fake_event = json!({
-                "eventId": "1234",
-                "userId": "123456",
-                "activity": $activity_type,
-                "timestamp": Utc::now(),
+        // Ignore our deployments & internal testing
+        let moose_internal = env::var("MOOSE_INTERNAL");
```

Comment:
Not sure if you know about this:

We have the ability to add settings at the cli level. In `apps/framework-cli/src/cli/settings.rs` The settings loaded there are using a hierarchy of env > files so that we could configure locally either env variables or configuration.

As I am writing this I now remember a discussion where we talked about letting the user opt out. 

For those reasons, having it in the cli settings with something like `enable_telemetry` that defaults to `true` might be good.

Reply:
Gotcha. There was a project config, but it makes more sense to move that to the cli level.